standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10567,6 +10587,16 @@ private StoredProcedure convertToStoredProc(String catName, MStoredProc proc) {
             proc.getSource());
   }
 
+  private Package convertToPackage(String catName, MPackage mPkg) {
```

Comment:
how about having this as `MPackage.toPackage()` ?

Reply:
fixed